staging/src/k8s.io/apimachinery/pkg/watch/watch.go

```diff
@@ -57,6 +58,7 @@ type Event struct {
 	// Object is:
 	//  * If Type is Added or Modified: the new state of the object.
 	//  * If Type is Deleted: the state of the object immediately before deletion.
+	//  * If Type is Bookmark: the object has set only ResourceVersion field.
```

Comment:
this needs more details... what is the type/structure of the object in a bookmark watch event? an instance of the type being watched? a status object?

Reply:
Added more details - PTAL if that seems good enough.